### PR TITLE
Update the internationalization of the build icon description for Korean

### PR DESCRIPTION
**Pull request by daeho-ro**
### Proposed changelog entries

Update the Korean internationalization for build icon descriptions.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] Appropriate autotests or explanation to why this change has no tests
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/jenkins/model/Jenkins/legend_ko.properties
Comment (daniel-beck) [inline core/src/main/resources/jenkins/model/Jenkins/legend_ko.properties:35]: For consistency, this should probably say
```suggestion
health-81plus=\uD504\uB85C\uC81D\uD2B8 \uAC74\uAC15\uC0C1\uD0DC\uAC00 80% \uCD08\uACFC\uC785\uB2C8\uB2E4. \uB9C8\uC6B0\uC2A4\uB97C \uD504\uB85C\uC81D\uD2B8 \uC544\uC774\uCF58\uC5D0 \uC62C\uB824\uB193\uC73C\uBA74 \uB354 \uC790\uC138\uD55C \uC124\uBA85\uC744 \uBCFC \uC218 \uC788\uC2B5\uB2C8\uB2E4.
```
At least the other entries use the round number. I don't know any Korean so that may well be wrong then (and need adjustments elsewhere).

**Review by daniel-beck — APPROVED** [review]
Looks good, thanks!

I don't think we have Korean speakers on the team. I checked the translations on Google Translate and they seemed reasonable.

File: core/src/main/resources/jenkins/model/Jenkins/legend_ko.properties
Comment (daeho-ro) [reply core/src/main/resources/jenkins/model/Jenkins/legend_ko.properties:35]: You are right. The original aim to fix is the number, it was 28% and now 81% which is also wrong :( I will fix it to 80%.

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
